File: how_many_words/solutions/how_many_words.java
class How_Many_Words{
  public static void main(String[] args) {
    char[] keep = {'1', '2', '3', '4', '5', '6', '7', '8', '9'};
    String word = "The dog jumps over the cat.";
    
    Integer the_first_answer = count_words(word);
    Integer the_second_answer = count_word(word);
    
    System.out.println(the_first_answer);
    System.out.println(the_second_answer);
    
  }
  
  public static int count_words(String phrase){
    String[] words = phrase.split(" ");
    return words.length;
  }
  
  public static int count_word(String phrase){
    int num_of_words = 1;
    char[] letters = phrase.toCharArray();
    for(char l : letters){
      if (l == ' '){
        num_of_words = num_of_words + 1;
      }
    }
    return num_of_words;
  }
  
}

File: how_many_words/solutions/how_many_words.py
'''
Given a phrase you need to return the number of words in the phrase.


"The dog jumps over the cat."
answer = 6
'''

def count_words(phrase):
    split_phrase = phrase.split(' ')
    return len(split_phrase)

def count_word(phrase):
    num_of_words = 1
    for c in phrase:
        if c == ' ':
            num_of_words += 1
    return num_of_words

print(count_words("The dog jumps over the cat."))
print(count_word("The dog jumps over the cat."))
print(count_words("cat"))
print(count_word("cat"))
